### ISSUE TITLE
Sneed's Damage not Correct

### ISSUE THREAD

**Issue by telummors** [dalaranwow/dalaran-wow#5337]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The Deadmines, NPC Sneed(phase 2 of fight) after Shredder phase.  Damage taken was 130-160 per melee swing while shredder form was 30ish damage. This was on my L25 Druid Bear Tank with 40% damage reduction.

**Expected behaviour**: Tell us what should happen instead
    Previous bosses at this level have also hit for around 30 damage, not 5x the damage.

**Steps to reproduce the problem**:

1.  Engage fight
2. Get to phase two
3. Watch Combat Log

**Include proofs for this behaviour**

![shredder phase](https://user-images.githubusercontent.com/37164519/37129052-04a049a6-224b-11e8-9cf3-6b6208529db1.PNG)
![sneedopswing](https://user-images.githubusercontent.com/37164519/37129053-04b0ab2a-224b-11e8-92a3-771761ea3fb7.PNG)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by HS-Disq** [comment]
I second this. I do not know how hard he is supposed to hit, but it's out of proportion compared to other bosses in this dungeon.

**Comment by Zarjofgt** [comment]
Honestly just buff the other bosses instead of nerfing him...
